src/redux/reducers/currentProjectReducer.ts

```diff
@@ -2,6 +2,7 @@ import _ from "lodash";
 import { ActionTypes } from "../actions/actionTypes";
 import { IProject } from "../../models/applicationState";
 import { AnyAction } from "../actions/actionCreators";
+import { AssetService } from "../../services/assetService";
```

Comment:
I don't think this line is required since you ended up not making the change here.